INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:07] "GET /search?s=ÔøΩÔøΩÔøΩ HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:07] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:10] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:10] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:11] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:11] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 34, in path_uploaded_pic
    raise TypeError("ÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩ ÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩ ÔøΩÔøΩÔøΩÔøΩÔøΩ {extension}")
TypeError: ÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩ ÔøΩÔøΩÔøΩÔøΩÔøΩÔøΩ ÔøΩÔøΩÔøΩÔøΩÔøΩ {extension}
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:05:20] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:06:29] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:06:29] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:06:38] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:06:38] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:07:01] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:07:01] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:07:01] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:11:52] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:11:52] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:11:56] "GET /search?s=–∫–æ—Ç HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:11:56] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:06] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:06] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:15] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:15] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:15] "GET /uploads/images/rc.jpg HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:56] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:56] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:58] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:12:58] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 34, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞ {extension}")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞ {extension}
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:13:16] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127. 0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:16:56] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:16:56] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:16:59] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:16:59] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 36, in path_uploaded_pic
    flash("error: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\helpers.py", line 391, in flash
    session["_flashes"] = flashes
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\sessions.py", line 97, in _fail
    raise RuntimeError(
RuntimeError: The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:17:10] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:20:28] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:20:28] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:20:32] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:20:32] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 40, in path_uploaded_pic
    flash("–§–æ—Ä–º–∞—Ç —Å–æ–æ—Ç–≤–µ—Ç—Å—Ç–≤—É–µ—Ç")
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\helpers.py", line 391, in flash
    session["_flashes"] = flashes
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\sessions.py", line 97, in _fail
    raise RuntimeError(
RuntimeError: The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:20:45] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 40, in path_uploaded_pic
    flash("–§–æ—Ä–º–∞—Ç —Å–æ–æ—Ç–≤–µ—Ç—Å—Ç–≤—É–µ—Ç")
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\helpers.py", line 391, in flash
    session["_flashes"] = flashes
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\sessions.py", line 97, in _fail
    raise RuntimeError(
RuntimeError: The session is unavailable because no secret key was set.  Set the secret_key on the application to something unique and secret.
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:20:55] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:18] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:18] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:25] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:25] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:32] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:32] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:32] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:35] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:35] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:39] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:39] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 35, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:22:55] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:26:03] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:26:03] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:26:07] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:26:07] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 35, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:26:17] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:27:03] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:27:03] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:27:05] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:27:05] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 35, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:27:15] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:29:50] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:29:50] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:29:53] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:29:53] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:root:–î–æ–±–∞–≤–ª—è–µ–º—ã–π —Ñ–∞–π–ª –Ω–µ –∫–∞—Ä—Ç–∏–Ω–∫–∞
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 17, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 36, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:30:04] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:35:35] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:35:35] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:35:57] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:35:58] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:root:–§–∞–π–ª –Ω–µ –Ω–∞–π–¥–µ–Ω.
Traceback (most recent call last):
  File "C:\Project\lesson12_project_source_v3\functions.py", line 9, in json_loaded_file
    with open('post.json', 'r', encoding='utf-8') as my_file:
FileNotFoundError: [Errno 2] No such file or directory: 'post.json'
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 26, in by_word_search
    if lowered_user_input in post["content"].lower().split(" "):
TypeError: string indices must be integers
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:36:01] "[35m[1mGET /search?s=–∫–æ—Ç HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:30] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:30] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:33] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:33] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:50] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:50] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:46:50] "[33mGET /–ù–µ%20–Ω–∞–π–¥–µ–Ω%20%20—Ñ–∞–π–ª HTTP/1.1[0m" 404 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:04] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:04] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:10] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:10] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:16] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:16] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:48:16] "[33mGET /–ù–µ%20–Ω–∞–π–¥–µ–Ω%20%20—Ñ–∞–π–ª HTTP/1.1[0m" 404 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:58:06] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:58:06] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:58:09] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:58:09] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    picture_url =  + pic_path
TypeError: bad operand type for unary +: 'str'
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 01:58:16] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:01:02] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:01:02] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:01:06] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:01:06] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:01:12] "POST /post HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:04:01] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:04:01] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:04:04] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:04:04] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:04:20] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:08:04] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:08:04] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:08:07] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:08:07] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:08:16] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:15:11] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:15:11] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:15:15] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:15:15] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:15:28] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:16:06] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:16:06] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:16:06] "[33mGET /–ù–µ%20–Ω–∞–π–¥–µ–Ω%20—Ñ–∞–π–ª HTTP/1.1[0m" 404 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:24:08] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:24:08] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:24:11] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:24:11] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:24:12] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:29:57] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:29:57] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:00] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:00] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:05] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:14] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:24] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:24] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:26] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:26] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:30:29] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:35:18] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:35:18] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:35:20] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:35:20] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:35:25] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:36:27] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:36:27] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:36:29] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:36:29] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:36:35] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:37:54] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:37:54] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:37:55] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:37:55] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:37:58] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:44:12] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:44:12] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:44:14] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:44:14] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise Exception("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
Exception: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 02:44:15] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 18:59:36] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 18:59:36] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 18:59:38] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 18:59:38] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 18:59:42] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:02:09] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:02:09] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:02:11] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:02:11] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:02:14] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:05:40] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:05:40] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:05:42] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:05:42] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:05:47] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:10:46] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:10:46] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:10:54] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:10:54] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:11:00] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:11:04] "GET /post HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:14] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:14] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:20] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:20] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:23] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:58] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:12:58] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:13:00] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:13:00] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:13:01] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:14:05] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:14:05] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:14:07] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:14:07] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 39, in path_uploaded_pic
    raise TypeError("–ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:14:08] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:31:28] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:31:28] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:31:35] "GET /search?s=–∫–æ—Ç HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:31:35] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:31:40] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:31:40] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1519, in full_dispatch_request
    return self.finalize_request(rv)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1538, in finalize_request
    response = self.make_response(rv)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1701, in make_response
    raise TypeError(
TypeError: The view function for 'blueprint_loader.uploaded_page' did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:32:01] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:31] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:31] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:33] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:33] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:58] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:59] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:37:59] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:41:45] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:41:45] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:41:52] "GET /search?s=–∫–æ—Ç HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:41:53] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:42:03] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:42:03] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:44:44] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:44:44] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:44:49] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:44:49] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:02] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:02] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:02] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:35] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:35] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:53] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:53] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:45:53] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:33] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:33] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:35] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:35] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:53] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:53] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:50:53] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:54:45] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:54:45] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:54:49] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:54:49] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:55:04] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:55:04] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:55:04] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:56:43] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:56:43] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 28, in by_word_search
    if lowered_user_input in post["content"].lower().split(" "):
TypeError: string indices must be integers
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:56:55] "[35m[1mGET /search?s=–ø—Ä–æ—Å—Ç–æ HTTP/1.1[0m" 500 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 28, in by_word_search
    if lowered_user_input in post["content"].lower().split(" "):
TypeError: string indices must be integers
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 19:57:20] "[35m[1mGET /search?s=—á–µ–ª–æ–≤–µ–∫ HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:00:35] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:00:35] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:00:45] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:00:45] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:00:48] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:00:48] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:01:12] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:01:12] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:01:12] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:04:38] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:04:38] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:04:45] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:04:45] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:05:04] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:05:04] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:05:04] "GET /uploads/images/rc.jpg HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:07:14] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:07:14] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:07:32] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:07:32] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:07:32] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:09:43] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:09:43] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:09:45] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:09:45] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:17] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:17] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:17] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:42] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:42] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:55] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:56] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:10:56] "GET /uploads/images/rc.jpg HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:13:50] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:13:50] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:13:54] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:13:54] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 28, in by_word_search
    if lowered_user_input in post["content"].lower().split(" "):
TypeError: string indices must be integers
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:14:01] "[35m[1mGET /search?s=–æ—á–µ–Ω—å HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:16:36] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:16:36] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 28, in by_word_search
    if lowered_user_input in post["content"].lower().split(" "):
TypeError: string indices must be integers
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:16:40] "[35m[1mGET /search?s=–∫–æ—Ç HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:53:29] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:53:29] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:53:34] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:53:34] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 30, in uploaded_page
    functions.post_add_to_json(make_post)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 58, in post_add_to_json
    posts.append(post)
AttributeError: 'dict' object has no attribute 'append'
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:53:47] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:55:13] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:55:13] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:55:16] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:55:16] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 30, in uploaded_page
    functions.post_add_to_json(make_post)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 58, in post_add_to_json
    posts.append(post)
AttributeError: 'dict' object has no attribute 'append'
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:55:33] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:56:36] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:56:36] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:56:41] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:56:41] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 30, in uploaded_page
    functions.post_add_to_json(make_post)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 58, in post_add_to_json
    posts.append(list(post))
AttributeError: 'dict' object has no attribute 'append'
INFO:werkzeug:127.0.0.1 - - [16/Mar/2022 20:56:52] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:00:17] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:00:17] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:00:19] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:00:19] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 32, in by_word_search
    raise TypeError("–ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!")
TypeError: –ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:00:25] "[35m[1mGET /search?s=–∫–æ—Ç HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:01:19] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:01:19] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 32, in by_word_search
    raise TypeError("–ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!")
TypeError: –ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:01:24] "[35m[1mGET /search?s=–∫–æ—Ç HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:47] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:47] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:50] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:50] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:57] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:58] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:06:58] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:07:00] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:07:00] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:07:07] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:07:07] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:07:07] "GET /uploads/images/rc.jpg HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:08:36] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:08:36] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 32, in by_word_search
    raise TypeError("–ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!")
TypeError: –ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:08:43] "[35m[1mGET /search?s=ppp HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:12:30] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:12:30] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 31, in by_word_search
    raise TypeError("–ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!")
TypeError: –ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:12:34] "[35m[1mGET /search?s=–∫–æ—Ç HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:12:53] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:12:53] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:13:11] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:13:11] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:13:11] "GET /uploads/images/cv.png HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:16:15] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:16:15] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:16:29] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:16:29] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:16:35] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:16:50] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:17:49] "POST /post HTTP/1.1" 200 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:22:49] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:22:49] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:22:54] "GET /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:22:54] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /post [POST]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\loader\views.py", line 19, in uploaded_page
    pic_path = functions.path_uploaded_pic(picture)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 40, in path_uploaded_pic
    raise TypeError("–ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞")
TypeError: –ö–∞—Ä—Ç–∏–Ω–∫–∞ –Ω–µ –Ω–∞–π–¥–µ–Ω–∞.–ù–µ–ø–æ–¥–¥–µ—Ä–∂–∏–≤–∞–µ–º—ã–π —Ñ–æ—Ä–º–∞—Ç —Ñ–∞–π–ª–∞
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:22:59] "[35m[1mPOST /post HTTP/1.1[0m" 500 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:23:42] "POST /post HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:23:42] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:23:42] "GET /uploads/images/rc.jpg HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:24:51] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:24:51] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\DNS\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Project\lesson12_project_source_v3\main\views.py", line 15, in search_page
    posts = by_word_search(s)
  File "C:\Project\lesson12_project_source_v3\functions.py", line 31, in by_word_search
    raise TypeError("–ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!")
TypeError: –ü–æ –∑–∞–ø—Ä–æ—Å—É –Ω–∏—á–µ–≥–æ –Ω–µ –Ω–∞–π–¥–µ–Ω–æ! –ü–æ–≤—Ç–æ—Ä–∏—Ç–µ –≤–≤–æ–¥ —Å –¥—Ä—É–≥–∏–º –∑–Ω–∞—á–µ–Ω–∏–µ–º!
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:24:58] "[35m[1mGET /search?s=–æ—á–µ–Ω—å HTTP/1.1[0m" 500 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:37] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:37] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:40] "GET /search?s=–æ—á–µ–Ω—å HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:40] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:53] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:53] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:57] "GET /search?s=–∫–æ—Ç HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:27:57] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:34:06] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:34:06] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:34:26] "GET /search?s=–æ—á–µ–Ω—å HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [17/Mar/2022 00:34:26] "[36mGET /static/styles/style.css HTTP/1.1[0m" 304 -
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
